.requirements

```diff
@@ -8,4 +8,4 @@ RESTY_OPENSSL_VERSION=1.1.1m
 RESTY_PCRE_VERSION=8.45
 LIBYAML_VERSION=0.2.5
 KONG_GO_PLUGINSERVER_VERSION=v0.6.1
-KONG_BUILD_TOOLS_VERSION=4.22.0
+KONG_BUILD_TOOLS_VERSION=master
```

Comment:
It seems that kong-build-tools-including-lmdb was already tagged, [many days ago](https://github.com/Kong/kong-build-tools/pull/426#issuecomment-1000779728)!
```suggestion
KONG_BUILD_TOOLS_VERSION=4.24.0
```